### DISCUSSION TITLE
gpiozero installed but not imported

### DISCUSSION THREAD

**Discussion by MMariam25** [gpiozero/gpiozero#968] [Q&A]
I am struggling with importing gpiozero. I have installed it using sudo apt install python3-gpiozero and tried with pip3 and python3 -mpip , i have done the update and upgrade. non of these worked for me. I always get ModuleNotFound: no module named 'gpiozero'

pleaase help

**Comment by bennuttall** [comment]
What OS are you using? Which version of Python? Did you install a different version of Python than the one provided by the OS? How are you running your code? From the terminal? From an IDE?
